hashintel/hash
# Add tooltips to section headers on entity editor

## 🌟 What is the purpose of this PR?

This PR adds tooltips to section headers explaining the purpose of each section. So users will have an idea about what each section is about.

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1201095311341924/1203199644802794/f) _(internal)_
- [Design](https://www.figma.com/file/gydVGka9FjNEg9E2STwhi2/HASH-App?node-id=4968%3A243891) _(internal)_


## ❓ How to test this?



1.  Checkout the branch / view the deployment
1.  Open entity editor
1.  Confirm section tooltips

## 📹 Demo

![Screen Shot 2022-10-25 at 14 05 59](https://user-images.githubusercontent.com/39553853/197757323-b8f3b44d-811c-45a3-8ae0-9cd5f186b9b1.png)
